src/css/controls/imports/modal.less

```diff
@@ -20,6 +20,6 @@
         width: 100%;
         max-height: none;
         max-width: none;
-        z-index: 1;
+        z-index: 2;
```

Comment:
needed?

Reply:
The time slider is shown on top of the shortcuts menu when its open on smaller players. Increasing the Z-index by 1 fixes it.